semantic_router/encoders/cohere.py

```diff
@@ -9,16 +9,23 @@
 class CohereEncoder(BaseEncoder):
     client: Optional[cohere.Client] = None
     type: str = "cohere"
+    input_type: Optional[str] = "search_query"
```

Comment:
@Siraj-Aizlewood, thanks for adding this. I have been wanting to do that.

Reply:
You're welcome :-)